### DISCUSSION TITLE
希望waline能接入Cravatar

### DISCUSSION THREAD

**Discussion by AkarinLiu** [walinejs/waline#464] [Q&A] [answered]
https://cravatar.cn/developers

**Comment by lizheming** [comment]
cravatar 好像和 gravatar 的 API 是兼容的，所以你应该直接直接配 avatarCDN 为 https://cravatar.cn/avatar/ 即可 https://waline.js.org/reference/client.html#avatarcdn

**Comment by Mister-Hope** [comment]
你现在可以通过 `AVATAR_PROXY` 自定义头像代理了。

**Comment by HaoTian22** [comment, marked as answer]
补充V2的配置方法：
Vercel里的`GRAVATAR_STR`环境变量设置为`https://cravatar.cn/avatar/{{mail|lower|trim|md5}}`
后面加这串是参考了cravatar的文档，防止大小写和空格误判
#1026
